cnd/src/network.rs

```diff
@@ -837,7 +833,7 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<comit::BehaviourOutEvent> for C
         let save_and_start_swap = async move {
             let swap = storage.load(local_swap_id).await?;
             save_swap_remote_data(&storage, swap, remote_data).await?;
-            spawn(&spawner, &storage, swap).await?;
+            spawn::spawn(&spawner, &storage, swap).await?;
```

Comment:
Why `spawn::spawn`? Seems pretty redundant to me :)

Reply:
Rust idiomatic ruling is to call functions that are from another module with the qualified path. We do not tend to do this but in this case it made the use statements less noisy so I favoured this syntax.

Reply:
> Rust idiomatic ruling is to call functions that are from another module with the qualified path.

Is this general statement actually true though? I never saw it as such a hard rule :)
I use qualified paths for functions if they add value to the name.